core/templates/components/question-directives/question-player/question-player.component.html

```diff
@@ -64,12 +75,14 @@ <h3 class="oppia-result-detail-header" translate="I18N_QUESTION_PLAYER_LEARN_MOR
               </div>
             </div>
           </div>
-          <div class="final-result">
+          <div class="final-result" tabindex="0"
+               [attr.aria-label]="'Your final result is: ' + finalCorrect.toFixed(2) +
+               ' correct answers out of total ' + allQuestions">
             <span class="score-final-description">Answers Correct:</span>
             <span class="score-final score-recieved">{{finalCorrect.toFixed(2)}}</span>
             <span class="score-final">/ {{allQuestions}}</span>
           </div>
-          <div class="score-calculation-text">This score is based on the number of hints you used and incorrect answers you entered during this session.</div>
+          <div class="score-calculation-text" tabindex="0">This score is based on the number of hints you used and incorrect answers you entered during this session.</div>
```

Comment:
Add the content `This score is based on the number of hints you used and incorrect answers you entered during this session.` to an I18N key since it is a learner facing component, and therefore this should be translated in all languages. ( You will have to add an I18N key to en.json file and what this key is for in qqq.json )

Ditto above on line 81 for `Answers Correct: `